certbot/cli.py

```diff
@@ -380,7 +380,9 @@ def _get_help_string(self, action):
     }),
     ("register", {
         "short": "Register for account with Let's Encrypt / other ACME server",
-        "opts": "Options for account registration & modification",
+        "opts": ("Options for account registration & modification"
+                 ". To register multi emails, separate by comma,"
+                 " ex: u1@example.com,u2@example.com"),
```

Comment:
We should put the documentation for multiple email addresses under the `--email` flag rather than here. The help text for this flag is defined in `certbot.interfaces.IConfig`.

Reply:
I'd also expand to "multi" to the full word "multiple".